test/base-tests/upload-to-aws.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+set -xeuo pipefail
+
+source /etc/os-release
+
+# Fedora 31's composer-cli doesn't support doing image uploads.
+if [[ $ID == fedora ]] && [[ $VERSION_ID == 31 ]]; then
+    echo "Fedora 31 does not support image uploads with composer-cli."
+    exit 0
```

Comment:
GNU autotools use "77" to skip tests. Is there a way to skip this test rather than make it succeed?

Reply:
Let's not care about Fedora 31 any more. We'll soon be interested in f33 instead.